src/github/pullRequestManager.ts

```diff
@@ -466,6 +472,28 @@ export class PullRequestManager implements vscode.Disposable {
 		});
 	}
 
+	getAllMentionableUsers(): IAccount[] | undefined {

```

Comment:
would you mind removing this and the corresponding events since it is unused?

Reply:
done